pkg/dispatcher/operator/operator.go

```diff
@@ -34,19 +34,21 @@ type Operator struct {
 	podsController         *workers.Controller
 	redisURL               string
 	redisClient            *redis.Client
+	redisSortedSetName     string
```

Comment:
This needs to end up to https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller.go#L40

Reply:
specific here: https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller.go#L176

Reply:
I dont need to use that redisSortedSetName in NewController(pkg/dispatcher/operator/workers/controllers.go), right?
I just create a flag in the main fail and in this file you are showing, then create new var in Controller struct and assign in the function like c.redisSortedSetName and that should be it, right?

Reply:
We hardcode the sorted set [here]( https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller.go#L176) and we need to change that bypassing the name via flags. Please check what it takes to do that.